src/read_write/s2.rs

```diff
@@ -101,3 +103,39 @@ where
         self.writers.get_mut(cell_id).unwrap()
     }
 }
+
+#[allow(dead_code)]
+pub fn s2_cloud_to_meta_proto(
```

Comment:
What about the `bounding_box` (and possibly `resolution`)? I think we need a way to fill these fields, too, otherwise it may make more sense to have a separate `S2Meta`, which would also clarify the difference between nodes and s2cells...

Reply:
indeed in the future they could be an interesting information about s2 cell points, Do we mean with bounding box the current one from the current inserted points, or the s2cell approximation into a bounding box? Resolution can be also expressed as a kind of average surface density in the current populated s2cell bounding box.

Reply:
Resolution doesn't make sense in combination with no PositionEncoding, so I think we should just leave it unset. The bounding box doesn't need to be exact, but we should probably have it since it's used in the client, right?

Reply:
here i am still in doubt what to do exactly here

Reply:
Do we even read the resolution anywhere?

Reply:
I just checked and I don't think so. So it's not too dangerous to leave it unset. As for bounding box, I'm fine with leaving it unset as well.